src/app-retrospective.c

```diff
@@ -180,6 +181,16 @@ i7_app_get_inform_command_line(I7App *self, const char *version_id, int format,
 		g_ptr_array_add(builder, g_strdup("-internal"));
 	g_ptr_array_add(builder, internal_path);
 
+	GSettings *prefs = i7_app_get_prefs(self);
+	g_autofree char *added_nest_dir = g_settings_get_string(prefs, PREFS_ADDED_NEST);
+	if (added_nest_dir) {
+		if ( (style == INFORM_9_1) || (style == INFORM_9_2) )
+			g_ptr_array_add(builder, g_strdup("-external"));
+		else if (style == INFORM_10_1) /* Should this be INFORM_10_2 only ? */
```

Comment:
To answer your question — it seems to work with 10.1.2, so `INFORM_10_1` seems appropriate here.